src/OpenAIGym.jl

```diff
@@ -10,7 +10,8 @@ import Reinforce:
 export
     gym,
     GymEnv,
-    test_env
+    test_env,
+    PyAny
```

Comment:
hmm, I don't think export this is a good idea.

Reply:
and seems there aren't definition of `test_env` in the whole codebase.

Reply:
Yeah, test_env should be removed (I didn't add it). 

PyAny I'm exporting for BlackJack's sake, but I guess we could drop it if we're doing the `envname=="Blackjack-v0"? PyArray: PyAny` change

Reply:
yeah, remove it
(https://github.com/JuliaML/OpenAIGym.jl/pull/11#discussion_r230585553)